docs/README.md

```diff
@@ -26,34 +27,54 @@
-   accessible in this repository).
-5. Fetch the previous release branch from `stolostron/gatekeeper`, create a new patch branch, and rebase on the new
-   release branch, accepting conflicts from the upstream changes:
+4. After the push, check the [Actions](https://github.com/stolostron/gatekeeper/actions) tab of the
+   repository to see whether there are new workflows that might need to be disabled (for example, if
+   it requires a token that is not accessible in this repository).
+5. Fetch the previous release branch from `stolostron/gatekeeper`, create a new patch branch, and
+   rebase on the new release branch, accepting conflicts from the upstream changes:
    ```shell
    git fetch ${STOLOSTRON} release-${PREV_RELEASE}
    git checkout ${STOLOSTRON}/release-${PREV_RELEASE}
    git checkout -b patch-release-${NEW_RELEASE}
    git rebase -X ours ${TAG}
    ```
-6. There now should be one or more commits on top from customizations in `stolostron`. Review the changes and squash the
-   commits into a single commit, removing irrelevant commits and updating the resulting description if necessary:
+   **NOTE:** As a sanity check, you can verify the new branch by visiting the GitHub comparison
+   page:
+   ```shell
+   echo "https://github.com/open-policy-agent/gatekeeper/compare/release-${NEW_RELEASE}...stolostron:gatekeeper:release-${NEW_RELEASE}"
+   ```
+6. There now should be one or more commits on top from customizations in `stolostron`. Review the
+   changes and squash the commits into a single commit, removing irrelevant commits and updating the
+   resulting description if necessary:
    ```shell
    git rebase -i ${TAG}
+   go mod vendor
+   git add .
+   git commit --amend --no-edit
    ```
-7. Push the patch branch to a development fork and open a PR against the new release branch (set `FORK` manually if you
-   have more than one development fork):
+7. Push the patch branch to a development fork and open a PR against the new release branch (set
+   `FORK` manually if you have more than one development fork):
    ```shell
    FORK="$(git remote -v | grep push | awk '!/(stolostron|open-policy-agent)/ {print $1}')"
    git push -u ${FORK} patch-release-${NEW_RELEASE}
    ```
-8. Once the PR is approved and merged: fetch the latest commits, clean the local tag, and tag and push the new release
-   in `stolostron/gatekeeper`:
-   ```shell
-   git fetch ${STOLOSTRON} release-${NEW_RELEASE}
-   git checkout ${STOLOSTRON}/release-${NEW_RELEASE}
-   git tag --delete ${TAG}
-   git tag -a ${TAG} -m ${TAG}
-   git push ${STOLOSTRON} ${TAG}
-   ```
+8. Once the PR is approved and merged: fetch the latest commits, clean the local tag, and tag and
+   push the new release in `stolostron/gatekeeper`:
+
+   - Re-set the new release version if needed:
+     ```shell
+     NEW_RELEASE_VERSION=X.Y.Z
+     ```
+   - Run these commands:
+
+     ```shell
+     STOLOSTRON="$(git remote -v | grep push | awk '/stolostron/ {print $1}')"
```

Comment:
What is grep put, which upstream is it?

Reply:
The output of `git remote -v` looks like this:
```
midstream       https://github.com/stolostron/gatekeeper-operator (fetch)
midstream       https://github.com/stolostron/gatekeeper-operator (push)
origin  https://github.com/dhaiducek/gatekeeper-operator (fetch)
origin  https://github.com/dhaiducek/gatekeeper-operator (push)
```
So it needs to filter out the duplicate entries.